### PR TITLE
[DOCS] Corrections to Document "How to request data from a Data Asset" with proper FDS implementation and integration test.

### PR DESCRIPTION
**Pull request by alexsherstinsky**
### Scope
* Repair of logic in document
* Insuring that logic of document is sound
* Adding integration test

### Remarks
* JIRA: DX-469/DX-441
* 

- [ ] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
- [x] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [x] Code is linted

    ```
    black .

    ruff . --fix
    ```
- [x] Appropriate tests and docs have been updated

For more details, see our [Contribution Checklist](https://docs.greatexpectations.io/docs/contributing/contributing_checklist), [Coding style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style), and [Documentation style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/docs_style).

After you submit your PR, keep the page open and **monitor the statuses of the various checks made by our continuous integration process at the bottom of the page. Please fix any issues that come up** and [reach out on Slack](https://greatexpectations.io/slack) if you need help. Thanks for contributing!

### CODE REVIEW

File: docs/docusaurus/docs/guides/connecting_to_your_data/fluent/batch_requests/how_to_request_data_from_a_data_asset.md [hunk 1]
```diff
@@ -59,18 +53,14 @@ You can create a dictionary of keys pulled from the `batch_request_options` tupl
...
-my_batch_request = my_asset.build_batch_request()
+```python name="tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/get_existing_data_asset_from_existing_datasource_pandas_filesystem_example.py my_batch_request"
 ```
 
 ### 5. Verify that the correct Batches were returned
 
 The `get_batch_list_from_batch_request(...)` method will return a list of the Batches a given Batch Request returns.  Because Batch definitions are quite verbose, it is easiest to determine what data the Batch Request will return by printing just the `batch_spec` of each Batch.
 
-```python title="Python code"
-batches = datasource.get_batch_list_from_batch_request(my_batch_request)
```
Comment (alexsherstinsky) [inline docs/docusaurus/docs/guides/connecting_to_your_data/fluent/batch_requests/how_to_request_data_from_a_data_asset.md:71]: This was the mistake (perhaps a typographical error) that was observed and the one that launched the revision, leading to the proposed changes.

File: docs/docusaurus/docs/guides/connecting_to_your_data/fluent/batch_requests/how_to_request_data_from_a_data_asset.md [hunk 2]
```diff
@@ -25,30 +32,19 @@ If you are using a Datasource that was created by using the block-config method
...
 
 ### 3. (Optional) Build an `options` dictionary for your Batch Request
 
 An `options` dictionary can be used to limit the Batches returned by a Batch Request.  Omitting the `options` dictionary will result in all available Batches being returned.
 
 The structure of the `options` dictionary will depend on the type of Data Asset being used.  The valid keys for the `options` dictionary can be found by checking the Data Asset's `batch_request_options` property.
 
-```python title="Python code"
-print(my_asset.batch_request_options)
+```python name="tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/get_existing_data_asset_from_existing_datasource_pandas_filesystem_example.py my_batch_request_options"
```
Comment (anthonyburdi) [inline docs/docusaurus/docs/guides/connecting_to_your_data/fluent/batch_requests/how_to_request_data_from_a_data_asset.md:47]:
```suggestion
```python 
 name="tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/get_existing_data_asset_from_existing_datasource_pandas_filesystem_example.py my_batch_request_options"
```

File: tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/get_existing_data_asset_from_existing_datasource_pandas_filesystem_example.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    order_by=["year", "month"],
+)
+
+# Python
+# <snippet name="tests/integration/docusaurus/connecting_to_your_data/fluent_datasources my_asset">
+my_asset = context.get_datasource("my_datasource").get_asset("my_asset")
+# </snippet>
+
+# Python
+# <snippet name="tests/integration/docusaurus/connecting_to_your_data/fluent_datasources my_batch_request_options">
```
Comment (anthonyburdi) [inline tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/get_existing_data_asset_from_existing_datasource_pandas_filesystem_example.py]:
```suggestion
# <snippet name="tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/get_existing_data_asset_from_existing_datasource_pandas_filesystem_example.py my_batch_request_options">
```

**Review by anthonyburdi — APPROVED** [review]
LGTM!

**Pull request merged**